Lib/profiling/sampling/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .collector import Collector
 from .pstats_collector import PstatsCollector
 from .stack_collector import CollapsedStackCollector
+from .gecko_collector import GeckoCollector
 from .string_table import StringTable
 
-__all__ = ("Collector", "PstatsCollector", "CollapsedStackCollector", "StringTable")
+__all__ = ("Collector", "PstatsCollector", "CollapsedStackCollector", "GeckoCollector", "StringTable")
```

Comment:
ℹ️ **Maintainability**

The `__all__` list has been updated to include `GeckoCollector`. Ensure that this addition is intentional and that `GeckoCollector` is meant to be part of the public API. If `GeckoCollector` is not intended for public use, consider removing it from `__all__`.